### DISCUSSION TITLE
S0 Counter works, but current power is always zero

### DISCUSSION THREAD

**Discussion by ctschach** [arendst/Tasmota#22676] [Support request]
I've a couple of S0 meters which I've integrated into Tasmota. Counting the actually impulses works fine and I see the meter value in kWh. This is done with:
`4,1-0:1.8.0*255(@1000,Zählerstand,kWh,ServerTotal,2
`

According to the documentation I should  also be able to get the current power by using this:
`4,1-0:1.7.0*255(@0.01667,Power,W,Server_Actual,0)
`

While the counter increases (I use aprox 200W), this always show zero.

Am I doing something wrong here?
Any way to debug this?
Any other way to calculate the current power usage?

```
>D

>B
->sensor53 r
tper=20

>M 4
+1,14,c,1,-25,Gas
+2,4,c,1,-40,Wasser
+3,4,c,1,-40,Studio
+4,4,c,1,-40,Server
1,1-0:1.8.0*255(@100,Zählerstand,m³,GasTotal,2
2,1-0:1.8.0*255(@10,Zählerstand,m³,WasserTotal,1
3,1-0:1.8.0*255(@1000,Zählerstand,kWh,StudioTotal,2
3,1-0:1.7.0*255(@0.01667,Power,W,Studio_Actual,0)
4,1-0:1.8.0*255(@1000,Zählerstand,kWh,ServerTotal,2
4,1-0:1.7.0*255(@0.01667,Power,W,Server_Actual,0)
;2,=h=========================
#
```

This runs on Tasmota 14.4.0.1 with scripting on an ESP32

**Comment by HofeBY2** [comment]
Ich bezweifle in Ihrem Beispiel, das alle S0 Counter richtig funktionieren,
da +2, +3, +4 den gleichen GPIO Port 4 verwenden.

Desweiteren sollte man KLammerebenen nicht nur öffnen, sondern auch schließen.

Umlaute würde ich auch nicht verwenden.

Die Zeile oberhalb "#" kann da deaktiviet auch gelöscht werden.

Ein Beispiel mit 5 Zähler finden Sie auch unter

github.com/HofeBY
